tools/armnn-script.js

```diff
@@ -0,0 +1,17 @@
+var flatbuffers = require('../third_party/flatbuffers/js/flatbuffers.js').flatbuffers;
```

Comment:
Can you remove this script. It shouldn't be needed and same can be accomplished in `armnn.js`.